source/wrangling.md

```diff
@@ -386,48 +242,74 @@ Data frame and vector types.
 > For example we can check the class of the Canadian languages data set,
 > `can_lang`, we worked with in the previous chapters and we see it is a `pandas.core.frame.DataFrame`.
 
-```{code-cell} ipython3
-:tags: [remove-cell]
-
-# The functions from the `tidyverse` package that we use often give us a
-# special class of data frame called a *tibble*. Tibbles have some additional \index{tibble}
-# features and benefits over the built-in data frame object. These include the
-# ability to add useful attributes (such as grouping, which we will discuss later)
-# and more predictable type preservation when subsetting. 
-# Because a tibble is just a data frame with some added features, 
-# we will collectively refer to both built-in R data frames and
-# tibbles as data frames in this book.
-
-# >  **Note:** You can use the function `class` \index{class} on a data object to assess whether a data
-# > frame is a built-in R data frame or a tibble. If the data object is a data
-# > frame, `class` will return `"data.frame"`. If the data object is a
-# > tibble it will return `"tbl_df" "tbl" "data.frame"`. You can easily convert
-# > built-in R data frames to tibbles using the `tidyverse` `as_tibble` function.
-# > For example we can check the class of the Canadian languages data set,
-# > `can_lang`, we worked with in the previous chapters and we see it is a tibble.
-```
 
 ```{code-cell} ipython3
 can_lang = pd.read_csv("data/can_lang.csv")
 type(can_lang)
 ```
 
-Lists, Series and DataFrames are basic types of *data structure* in Python, which
-are core to most data analyses. We summarize them in 
-{numref}`tab:datastructure-table`. There are several other data structures in the Python programming 
-language (*e.g.,* matrices), but these are beyond the scope of this book.
+### Data structures in Python
```

Comment:
This may not need its own title, but I found it helpful. Feel free to remove if you think it is best to flow from the content about DataFrames above